services/workflows-service/src/case-management/case-management.service.ts

```diff
@@ -66,7 +66,7 @@ export class CaseManagementService {
     const isValid = validate(entity);
 
     if (!isValid) {
-      throw ValidationError.fromAjvError(validate.errors!);
+      throw BadValidationException.fromAjvError(validate.errors!);
```

Comment:
Is the rename due to the naming conflict with class-validator?

Reply:
yeah, i could use 'as' which its confusing to use it and rename this class name in some places